### ISSUE TITLE
BUG - sev-3 - All - Need active state color for all components touched during home screen personalization

### ISSUE THREAD

**Issue by TKDickson** [department-of-veterans-affairs/va-mobile-app#9302]
## What happened?

We need an active state color for all of these components (missed during HSP):

- Veterans Crisis Line
- Activity section buttons
- Veteran Status Card 'nametag' button
- Announcements banner
- Link Rows, and
- Large Nav buttons on category screens

Active state = color change when tapped, like the preferred name row here:
<img width="428" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBd0pFQmc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--c51d22370e7b605776f860051a91386eb16a6d9f/image.png" alt="image.png" />

## Specs:

- Device:
- OS:
- User Account:
- Accessibility State:

## Desired behavior

I confirmed all of the active state colors with Lauren.

We should be matching the list (aka the preferred name screenshot above) for:
- Veteran Status Card
- Announcements
- Link Row, and
- Category nav buttons

And we should be following the [design system buttons documentation](https://www.figma.com/design/Zzt8z60hCtdEzXx2GFWghH/%F0%9F%93%90-Component-Library---Design-System---VA-Mobile?node-id=224-548&t=drwDPH5Q4XXEkYzu-0) for:
- VCL, and
- Activity buttons

## Bug Severity - BE SURE TO ADD THE SEVERITY LABEL
 
See [Bug Tracking](https://department-of-veterans-affairs.github.io/va-mobile-app/docs/QA#issue-severity) for details on severity levels

- Impact: Low 
- Frequency: Low 

- 3 - Low
- 2 - High
- 1 - Critical

## Screen shot(s) and additional information

<details>
  <summary>Full JSON response for services related to issue (expand/collapse)</summary>
  

</details>

## Ticket Checklist
- [ ] Steps to reproduce are defined
- [ ] Desired behavior is added
- [ ] Labels added (front-end, back-end, global, Health, relevant feature, accessibility, etc)
- [ ] Estimate of 1 added (for front-end tickets)
- [ ] Added either to the relevant feature epic (if found during new feature implementation), or the relevant team's bug epic ([Global](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8302), [Health & Benefits](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8303), [API](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8304), [QART](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8305))

**Comment by TKDickson** [comment]
Confirmed active colors match design in both light and dark mode, Android and iOS. Since it involved component refactoring I also did a [smoke test of all supported assistive tech](https://dsvavsp.testrail.io/index.php?/tests/view/1262717&group_by=cases:section_id&group_order=asc&group_id=4180) and there was no change in or loss of functionality across these components.

**Comment by TKDickson** [comment]
Doesn't require further testing in develop; closing
